README.md

```diff
@@ -1,27 +1,35 @@
-<img src="https://user-images.githubusercontent.com/57671048/103117235-80cb3200-468f-11eb-93ca-766e6f42f5b1.jpg" height="200">
+#  Facial Emotion Investigator :
+
+<img src=https://www.apa.org/images/PSA-2011-05-matsumoto-fig1_tcm7-115934_w1024_n.jpg height="200">
 
 [![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/)
+<img src="https://img.shields.io/badge/CONTRIBUTORS-1-green">
+<img src="https://img.shields.io/badge/FORKS-4-blue">
+<img src="https://img.shields.io/badge/STARS-9-blue">
+<img src="https://img.shields.io/badge/ISSUES-10%20OPEN-yellow">
 [![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/Naereen/StrapDown.js/blob/master/LICENSE)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
 
-# :sunglasses: Facial Expression Recognition Classifier Model 
+## :sunglasses:  About The Project:
+
+# Facial Expression Recognition Classifier Model
 
-A Facial Expression Recognition Classifier Model that takes
-**Real-time Video input** and predicts the **Emotion of users** present in front of
-Webcam. It also gives **Graphical Visualization** of Expressions when we feed in an
-Image via Web Cam or manually !
+Facial expression for emotion detection has always been an easy task for humans, but achieving the same task with a computer algorithm is quite challenging. With the recent advancement in computer vision and machine learning, it is possible to detect emotions from images.In this project,we propose a novel technique called facial emotion recognition using convolutional neural networks,python and flask. Facial expressions are the vital identifiers for human feelings, because it corresponds to the emotions. Most of the times (roughly in 55% cases) [1], the facial expression is a nonverbal way of emotional expression, and it can be considered as concrete evidence to uncover whether an individual is speaking the truth or not [2].
```

Comment:
Remove [1] and [2] Numbering in the Last Statement

Reply:
I have changed it